Tests/TensorFlowTests/TensorAutoDiffTests.swift

```diff
@@ -630,7 +645,7 @@ final class TensorAutoDiffTests: XCTestCase {
   }
 
   func testSideEffects() {
-    let foo: @differentiable (Tensor<Float>) -> Tensor<Float> = { x in
+    let foo: @differentiable(Tensor<Float>) -> Tensor<Float> = { x in
```

Comment:
Could you please revert this unrelated change? I think the original formatting `@differentiable (Tensor<Float>) -> Tensor<Float>` is correct.

Reply:
Fixed. `swift-format` did this change.
Perhaps my `swift-format` installation is wrong. Is there documentation describes how to install `swift-format` for S4TF?